crates/bevy_scene/src/scene_spawner.rs

```diff
@@ -34,11 +34,11 @@ pub struct SceneSpawner {
 
 #[derive(Error, Debug)]
 pub enum SceneSpawnError {
```

Comment:
It might be better to just reword the errors to not be questions. One way would be to add documentation to the error that suggests that the scene might be still loading, and just having the error be `scene does not exist`.